conans/assets/templates/new_v2_cmake.py

```diff
@@ -204,21 +204,23 @@ def test(self):
 test_main = """#include "{name}.h"
 
 int main() {{
-    {name}();
+    {symbol_name}();
 }}
 """
 
 
-def get_cmake_lib_files(name, version, package_name="Pkg"):
+def get_cmake_lib_files(name, version, package_name="Pkg", symbol_name="pkg"):
```

Comment:
The default ``symbol_name="pkg"`` is dangerous. It shouldn't have a default.

Reply:
I didn't want to break the API of this function. Should I add it as a mandatory parameter before package_name (I wondered this has a default anyway)?

Reply:
Thanks for the fast feedback anyway.

Sorry, I cannot find where the rendering context is defined in the develop2 branch.

Is this implemented yet? The template looks a bit WIP there, containing

```class {{class_name or name}}Recipe(ConanFile):```

and I cannot find the definition of `class_name` anywhere.

Maybe these variables should be renamed according to their formatting like upper_camel_case_name and lower_camel_case_name? This would be more verbose than object_name.

Reply:
> I didn't want to break the API of this function. Should I add it as a mandatory parameter before package_name (I wondered this has a default anyway)?

It is purely internal function, probably called just once in code. No probe to change it.

Reply:
> Is this implemented yet? The template looks a bit WIP there, containing

Yes, the ``conan new cmake_lib -d name=pkg -d version=0.1`` command is fully operational in alpha3 and in develop2 branch. 
The templates are jinja templates, ``-d class_name=Something`` will dynamically inject it.

Reply:
Ah I see, I need to manually inject the definitions. It was filled automatically in Conan 1.x, but it makes sense to make this explicit.

Reply:
Yes, we are moving the whole "new" functionality to pure jinja templates, and a thin command line layer.

Reply:
OK have changed the commit. It's not really worth the effort any more 🤷 I'll close the corresponding docs PR